fluidBuild.config.cjs

```diff
@@ -59,19 +59,19 @@ module.exports = {
 				"webpack-dev-server",
 			],
 		},
-    // These packages are independently versioned and released, but we use pnpm workspaces in single packages to work
-    // around nested pnpm workspace behavior. These packages are not checked for the preinstall script that standard
-    // pnpm workspaces should have.
-    pnpmSinglePackageWorkspace: [
-      "@fluid-tools/api-markdown-documenter",
-      "@fluid-tools/benchmark",
-      "@fluidframework/build-common",
-      "@fluidframework/common-definitions",
-      "@fluidframework/common-utils",
-      "@fluidframework/eslint-config-fluid",
-      "@fluidframework/protocol-definitions",
-      "@fluidframework/test-tools",
-    ],
+		// These packages are independently versioned and released, but we use pnpm workspaces in single packages to work
+		// around nested pnpm workspace behavior. These packages are not checked for the preinstall script that standard
+		// pnpm workspaces should have.
+		pnpmSinglePackageWorkspace: [
+			"@fluid-tools/api-markdown-documenter",
+			"@fluid-tools/benchmark",
+			"@fluidframework/build-common",
+			"@fluidframework/common-definitions",
+			"@fluidframework/common-utils",
+			"@fluidframework/eslint-config-fluid",
+			"@fluidframework/protocol-definitions",
+			"@fluidframework/test-tools",
+		],
```

Comment:
This looks like a good fix: I'm guessing an improved config from the updated build tools detected/fixed this?

Reply:
I think so, yes.